views/admin/assignments.html

```diff
@@ -644,6 +645,17 @@ <h5 style="text-align: center">Search Question Bank</h5>
         },
     });
 });
+  
+var totalPointsDiv = document.getElementById('totalPoints');
```

Comment:
I like this, but I would rather have this added to admin.js.

I'm slowly trying to get the javascript out of the html files and into js files to make it easier to maintain.